### PR TITLE
Tas-22 create a signup page

### PR DESCRIPTION
**Pull request by kajal1106**
🚀 Implemented and refined the Signup page

In this pull request, I've done the following work on the Signup page. Here's a summary of the changes:

- ✨ Added a new registration flow for a smoother user experience
- 🛠 Refactored existing code structure for better maintainability
- 🚮 Removed unused activities and optimized resource files
- 🎨 Improved UI/UX elements, ensuring a consistent design
- 🧪 Conducted thorough testing, addressing potential edge cases

Thank you! 🙌

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,9 +14,16 @@
...
         tools:targetApi="31">
-
+<!--        <activity-->
+<!--            android:name=".ui.activity.SignupActivity.SignupActivity"-->
+<!--            android:exported="true">-->
+<!--            <intent-filter>-->
+<!--                <action android:name="android.intent.action.MAIN" />-->
+<!--                <category android:name="android.intent.category.LAUNCHER" />-->
+<!--            </intent-filter>-->
+<!--        </activity>-->
```
Comment (Agata-Duda) [inline app/src/main/AndroidManifest.xml:24]: Does commented code need to be pushed to main? :)

File: local.properties [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
+## This file must *NOT* be checked into Version Control Systems,
 # as it contains information specific to your local configuration.
 #
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-sdk.dir=/Users/psmattas/Library/Android/sdk
\ No newline at end of file
+#Fri Oct 27 10:59:25 IST 2023
+sdk.dir=/Users/nainsha/Library/Android/sdk
```
Comment (Agata-Duda) [inline local.properties:8]: Don't think this file should be changed as per header comment *Do not check into version control* Potentially we can delete this file from the PR? Open do discuss :)

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,22 +1,33 @@
...
+<!--    <com.google.android.material.textview.MaterialTextView-->
+<!--        android:layout_width="match_parent"-->
+<!--        android:layout_height="match_parent"-->
+<!--        android:text="@string/app_name"-->
+<!--        android:gravity="center"-->
+<!--        android:textSize="50sp"-->
+<!--        app:layout_constraintBottom_toBottomOf="parent"-->
+<!--        app:layout_constraintEnd_toEndOf="parent"-->
+<!--        app:layout_constraintStart_toStartOf="parent"-->
+<!--        app:layout_constraintTop_toTopOf="parent" />-->
```
Comment (Agata-Duda) [inline app/src/main/res/layout/activity_main.xml:31]: Do we need the commented code here, if not we can also delete it.

**Review by Agata-Duda — CHANGES_REQUESTED** [review]
Just some minor comments in relation to commented code, code structure looks good and functionality too !

File: local.properties (hunk 2 above)
Comment (kajal1106) [reply local.properties:8]: Yeah this file has accidentally been pushed, I'll remove it from the codebase

Comment (Agata-Duda) [reply local.properties:8]: Great thank you

**Review by psavarmattas — APPROVED** [review]
LGTM 🫡

**Review by sandeshkulk — COMMENTED** [review]
Looks good to me.

**Pull request merged**